src/Tests/InstrEngineTests/InstrEngineTests/ProfilerHelpers.cs

```diff
@@ -174,9 +188,25 @@ public static void LaunchAppUnderProfiler(TestParameters parameters, string test
             }
 
             psi.EnvironmentVariables.Add(TestOutputEnvName, PathUtils.GetAssetsPath());
-            psi.EnvironmentVariables.Add(
-                is32bitTest ? HostConfig32PathEnvName : HostConfig64PathEnvName,
-                Path.Combine(PathUtils.GetAssetsPath(), string.Format(CultureInfo.InvariantCulture, "NaglerInstrumentationMethod_{0}.xml", bitnessSuffix)));
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                psi.EnvironmentVariables.Add(
+                    is32bitTest ? HostConfig32PathEnvName : HostConfig64PathEnvName,
+                    Path.Combine(PathUtils.GetAssetsPath(), string.Format(CultureInfo.InvariantCulture, "NaglerInstrumentationMethod_{0}.xml", bitnessSuffix)));
+            }
+#if NETCOREAPP
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD))
+            {
+                Console.Error.WriteLine($"{OSPlatform.FreeBSD:G} is not supported.");
+            }
+#endif
+            else
+            {
+                psi.EnvironmentVariables.Add(
+                    HostConfig64PathEnvName,
+                    Path.Combine(PathUtils.GetAssetsPath(), "LinuxNaglerInstrumentationMethod.xml"));
+            }
```

Comment:
nit: I think this could technically be in the NETCOREAPP as well.

Reply:
true it doesn't change the behavior so I'll leave it as-is.